app/src/main/java/ch/benediktkoeppel/code/droidplane/HorizontalMindmapView.java

```diff
@@ -604,7 +616,50 @@ private int getVisiblePixelOfLeftmostColumn() {
 
         return numVisiblePixelsOnColumn;
     }
+    
+    /** Show a dialog to input the search string and fires the search. */
+    void startSearch() {
+        AlertDialog.Builder alert = new AlertDialog.Builder(getContext());
+        EditText input = new EditText(getContext());
+        input.setText(lastSearchString, TextView.BufferType.EDITABLE);
+        alert.setView(input);
+        alert.setPositiveButton("OK", (dialog, which) -> search(input.getText().toString()));
```

Comment:
I suggest to rename "OK" to "Search" to make it clear what it's doing

Reply:
Agreed, I think this is left from the copy & paste from StackOverflow...